client/deposits/list/index.js

```diff
@@ -51,7 +51,8 @@ const columns = [
 ];
 
 export const DepositsList = () => {
-	const { deposits, isLoading } = useDeposits( getQuery() );
+	const { deposit } = useInstantDeposit();
+	const { deposits, isLoading } = useDeposits( getQuery(), deposit );
```

Comment:
It feels wrong that the React component needs to care about the "plumbing" like this. Isn't there a way to express this dependency entirely inside the data layer?

Reply:
Thanks for pair programming and helping with this, it's appreciated!